tests/test_functions.py

```diff
@@ -37,6 +37,19 @@ def dataframe():
     return df
 
 
+@pytest.fixture
+def dataframe_floats():
```

Comment:
I'm wondering if we can remove some duplication here.

If you change the `def dataframe()` fixture such that it returns all of the float values below, do tests pass? If so, I think there will be less code to maintain by making the `dataframe()` fixture return the contents of the `dataframe_floats()` fixture instead.

Reply:
I changed the dataframe fixture to include floats. Just had to change a few tests to reflect.